packages/postcss-merge-longhand/src/__tests__/borders.js

```diff
@@ -154,6 +154,20 @@ test(
     'hr{display:block;height:1px;border:0;border-top:1px solid #ddd}'
 );
 
+test(
+    'Should not mangle borders (#579) (1)',
+    processCSS,
+    'h1{border-bottom:none;border-color:red}',
+    'h1{border-bottom:0;border-color:red}',
```

Comment:
`border: none` !== `border: 0` https://stackoverflow.com/questions/2922909/should-i-use-border-none-or-border-0

Reply:
It is no safe